TravelAgent/TravelAgent/DataContract/Requests/RegisterAccountRequest.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace TravelAgent.DataContract.Requests
+{
+    public class RegisterAccountRequest
+    {
+        [Required]
+        public string UserName { get; set; }
```

Comment:
Gal galim tik e-mail naudoti, be username? Tiesiog, dėl aiškumo.

Reply:
Darbuotojai vistiek dažnai turi su jais surištą darbo mail'ą

Reply:
galiu panaikinti